grpc build is broken

https://oss-fuzz-build-logs.storage.googleapis.com/log-f3afc45e-5495-481c-94fa-50b636329716.txt

These appear to be the relevant lines:
Step #3: test/core/http/request_fuzzer.cc:36:59: error: use of undeclared identifier 'nullptr'
Step #3:   GRPC_ERROR_UNREF(grpc_http_parser_parse(&parser, slice, nullptr));
Step #3:                                                           ^
Step #3: 2 warnings and 1 error generated.
Finished Step #3
ERROR
ERROR: build step "gcr.io/oss-fuzz/grpc" failed: exit status 1

[request_fuzzer.cc](https://github.com/grpc/grpc/blob/master/test/core/http/request_fuzzer.cc) does not include `<cstddef>`, which is where `nullptr` is declared. Maybe this needs to be explicitly included.

--- a/projects/grpc/build.sh
+++ b/projects/grpc/build.sh
@@ -62,12 +62,9 @@ CXXFLAGS="${CXXFLAGS} -Iinclude -I. -stdlib=libc++"
 
 for file in $FUZZER_FILES; do
   fuzzer_name=$(basename $file .cc)
-  fuzzer_object="${file::-1}o"
   echo "Building fuzzer $fuzzer_name"
-  $CC $CFLAGS \
-    $file -c -o $fuzzer_object 
   $CXX $CXXFLAGS \
-    $fuzzer_object -o $OUT/$fuzzer_name \
+    $file -o $OUT/$fuzzer_name \
     -lFuzzingEngine ${FUZZER_LIBRARIES}
 done
 

